app/src/main/java/com/github/polypoly/app/menu/ProfileActivity.kt

```diff
@@ -79,7 +86,8 @@ class ProfileActivity : ComponentActivity() {
         val mContext = LocalContext.current
         Column(
             modifier = Modifier
-                .padding(all = 30.dp),
+                .padding(all = 30.dp)
+                .background(color = Color.White),
```

Comment:
You should avoid things like that, you created a palette so it's better to use it. What if we would like to change our background color everywhere. You could use `MaterialTheme.colors.background`